src/algebra/monoid_algebra/basic.lean

```diff
@@ -1542,11 +1542,11 @@ lemma lift_unique' (F : add_monoid_algebra k G →ₐ[k] A) :
 /-- Decomposition of a `k`-algebra homomorphism from `monoid_algebra k G` by
 its values on `F (single a 1)`. -/
 lemma lift_unique (F : add_monoid_algebra k G →ₐ[k] A) (f : monoid_algebra k G) :
-  F f = f.sum (λ a b, b • F (single a 1)) :=
-by conv_lhs { rw lift_unique' F, simp [lift_apply] }
+  F f = f.sum (λ a b, b • F (of k G a)) :=
+by { conv_lhs { rw lift_unique' F, simp [lift_apply], }, refl, }
 
 lemma alg_hom_ext_iff {φ₁ φ₂ : add_monoid_algebra k G →ₐ[k] A} :
-  (∀ x, φ₁ (finsupp.single x 1) = φ₂ (finsupp.single x 1)) ↔ φ₁ = φ₂ :=
+  (∀ x, φ₁ (of k G x) = φ₂ (of k G x)) ↔ φ₁ = φ₂ :=
```

Comment:
This feel slike it will be irritating, because it introduces `x : multiplicative G` instead of `x : G` like it used to. Changing from `of` to `of'` would fix this.